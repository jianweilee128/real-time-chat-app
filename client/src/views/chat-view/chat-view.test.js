import React from "react";
import { shallow } from "enzyme";

import ChatView from "./chat-view";

describe("ChatView component", () => {
  let wrapper;

  beforeEach(() => {
    const mockProps = {
      messageList: [
        {
          message: "Hey",
          sender: "JohnDoe",
          timestamp: "10.37pm",
        },
        {
          message: "Hi",
          sender: "JaneDoe",
          timestamp: "10.38pm",
        },
      ],
    };

    wrapper = shallow(<ChatView.WrappedComponent {...mockProps} />);
  });

  it("should render ChatView component", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
