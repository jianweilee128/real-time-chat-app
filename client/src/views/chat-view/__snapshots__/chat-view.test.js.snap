// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ChatView component should render ChatView component 1`] = `ShallowWrapper {}`;
